element.go

```diff
@@ -126,8 +126,8 @@ func newElement(t tag.Tag, data interface{}) (*Element, error) {
 	}, nil
 }
 
-func mustNewElement(t tag.Tag, data interface{}) *Element {
-	elem, err := newElement(t, data)
+func MustNewElement(t tag.Tag, data interface{}) *Element {
```

Comment:
let's leave this unexported if only used within this package, would rather not clutter the exported namespace of the package. Callers can just use NewElement with the error, or write a small must new element shim if needed on their end, doesn't necessarily need to be part of our API I think. As much as possible I'd like to have a minimal exported API, but let me know if you have some good use cases or thoughts otherwise!

Reply:
Sounds good, I only made this exported since the old api utilized the MustNewElement method, but I think just using NewElement and handling the error will be fine.